app/views/content/non-uk-teachers/non-uk-qualifications.md

```diff
@@ -110,9 +110,9 @@ If you have not passed an English language test like this, or do not have qualif
 
 ## If you do not have a bachelor's degree
 
-If you do not have a degree, you can apply to study a 3 to 4 year [undergraduate course in England](https://www.ucas.com/postgraduate/teacher-training/applying-teacher-training/find-teacher-training-programmes) which combines a teaching degree with [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts). When you complete your studies, you’ll be able to apply for jobs as a teacher in England.
+If you do not have a degree, you can apply for an [undergraduate course in England on the UCAS website](https://www.ucas.com/postgraduate/teacher-training/applying-teacher-training/find-teacher-training-programmes). These ungergraduate courses are 3 to 4 years and combine a teaching degree with [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts). When you complete your studies, you’ll be able to apply for jobs as a teacher in England.
```

Comment:
Why do we go the GOV.UK QTS page here instead of our page?

Reply:
Typo `ungergraduate`

Reply:
Re why we go to the GOV.UK page - suspect this page was written before we had our QTS page. I agree it doesn't make sense here and we should link to our QTS page